src/components/consul/selectors/ImageSelector.js

```diff
@@ -205,14 +237,15 @@ ImageSelector.propTypes = {
   control: PropTypes.object,
   field: PropTypes.object,
   imageId: PropTypes.string,
+  isConsul: PropTypes.bool,
   isVolunteer: PropTypes.bool,
   item: PropTypes.object
 };
 
 const styles = StyleSheet.create({
   image: {
-    height: imageHeight(imageWidth() * 0.6),
-    width: imageWidth() * 0.6
+    height: imageHeight(imageWidth() * 0.8),
+    width: imageWidth() * 0.8
```

Comment:
i will revert this to result in 

![image](https://user-images.githubusercontent.com/1942953/208416022-ebbfd10d-badd-493d-bf94-5fe5ce72ee59.png)

Reply:
At least the delete button can be pressed. 😂👍